skydoc/common.py

```diff
@@ -25,6 +25,12 @@
 OUTPUTS_HEADING = "Outputs:"
 
 
+class InputError(Exception):
+  """Exception raised for errors in the input to Skydoc."""
```

Comment:
Just pass (http://stackoverflow.com/questions/1319615/proper-way-to-declare-custom-exceptions-in-modern-python)